streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -342,4 +342,20 @@ private RecordQueue createRecordQueue(TopicPartition partition, SourceNode sourc
         }
     }
 
+    @SuppressWarnings("unchecked")
+    public <K, V> void forward(K key, V value, String childName) {
+        ProcessorNode thisNode = currNode;
+        try {
+            for (ProcessorNode childNode : (List<ProcessorNode<K, V>>) thisNode.children()) {
```

Comment:
@kawamuray Thanks for the comment. I am expecting number of children to be very small (<5) in which case iterating will be fine.